### ISSUE TITLE
CHAINOS SOLUTION- IOS DEVELOPER- Hanoi- Fulltime

### ISSUE THREAD

**Issue by tulitran** [awesome-jobs/vietnam#2579]
## Location
* Hanoi, Vietnam
* Fulltime

## Salary Expectation
* Up to 30M

## Requirements
* Have at least 1 year of iOS programming experience
* Master the knowledge of OOP
* Experience working with Objective-C/Swift
* Understand and work with MVC and MVVM models
* Experience working with Firebase, SocketIO, ReactiveObjC...
* Use the RESTful API to communicate with the server.
* Capable of working independently, as a team.
* Be responsible for work, work under high pressure.

### Ideally

* Knowing more Android or Flutter

### Responsibilities
* Build top products with iOS on a large user database platform; working with a professional and experienced team;
* Directly accompany the Head of Department, Project Management, Backend, and UX team in a flexible and responsive environment to increase the progress of product distribution;
* Directly participate in the project implementation process
* Great contribution to mobile programming and technical expertise of the entire organization

### WORKING HOUR - WORKING PLACE
* Working time: 8:30 AM - 6:00 PM from Monday to Friday.
* Location: 8th floor, HL Tower - 82 Duy Tan Street, Cau Giay district, Ha Noi City.

### BENEFITS:
* The promotion roadmap is clear, Review salary 2 times/year. Trained to be a Team Leader/ Technical Leader.
* Have onsite opportunities at JP and Korea
* Enjoy the regimes on social insurance, health insurance, unemployment insurance, and other regimes as prescribed by the Company and the Labor Law.
* The company buys more PVI health insurance packages with many benefits and is used in high-end hospitals such as Hong Ngoc, Viet France ... Especially family members such as parents, wives, children ... can participate in this insurance package at a preferential rate.
* Many extracurricular activities, life culture, sports, team building, relax ... organized.

### Contact
* Email: linhttt@chainos.vn
* Phone/ Zalo: 0972.770.924
* Facebook: fb.com/ThLinhHH/

**Comment by high400** [comment]
Yeah, i'm currently working with React Native + Android here. Guess it's my lucky day
